src/screens/PluginStore/UploadPluginModal.tsx

```
@@ -52,98 +42,45 @@
-          );
+        if (structure.hasApiFolder) {
+          filesList.push('api/');
+          structure.apiFiles?.forEach((file) => {
+            filesList.push(`    ${file}`);
+          });
         }
 
-        setManifest(manifestData);
+        setPluginFiles(filesList);
+        setPluginStructure(structure);
+        setManifest(structure.adminManifest || structure.apiManifest!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)